code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -352,44 +342,31 @@ Please contact me on #coderbus IRC. ~Carnie x
 			drop_l_hand()
 			drop_r_hand()
 
-	src.update_hair()
-	src.update_mutant_bodyparts()
+	update_hair()
+	update_mutant_bodyparts()
 
 	apply_overlay(SUIT_LAYER)
 
 
 /mob/living/carbon/human/update_inv_pockets()
 	if(l_store)
-		l_store.screen_loc = ui_storage1	//TODO
-		if(client && hud_used)
+		if(client && hud_used && hud_used.hud_shown)
 			client.screen += l_store
+			l_store.screen_loc = ui_storage1
 	if(r_store)
-		r_store.screen_loc = ui_storage2	//TODO
-		if(client && hud_used)
+		if(client && hud_used && hud_used.hud_shown)
 			client.screen += r_store
+			r_store.screen_loc = ui_storage2

```

Comment:
All these X.screen_loc = Y" are moved so the object doesn't show up on our hud if our hud is not used or shown(toggled with f12).